### PR TITLE
Cookie initialy baked by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "develop",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-plugin-nornir.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-nornir",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-nornir",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "@smith-ntc",
        "full_name": "Network to Code, LLC",
        "email": "info@networktocode.com",
        "github_org": "nautobot",
        "plugin_name": "nautobot_plugin_nornir",
        "verbose_name": "Nautobot Nornir Plugin",
        "plugin_slug": "nautobot-plugin-nornir",
        "project_slug": "nautobot-plugin-nornir",
        "repo_url": "https://github.com/nautobot/nautobot-plugin-nornir",
        "base_url": "plugin-nornir",
        "min_nautobot_version": "2.0.0",
        "max_nautobot_version": "2.9999",
        "camel_name": "NautobotPluginNornir",
        "project_short_description": "Nautobot Nornir Plugin",
        "model_class_name": "None",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/plugin-nornir/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/opt/ntc/drift-manager/outputs/.cookiecutters/cookiecutter-nautobot-app/nautobot-app",
        "_checkout": "develop"
    },
    "base_branch": "develop",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.UPDATE_OR_CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ],
    "baked_commit_ref": "",
    "draft": true
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "setup-cookie-nornir.json",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "develop",
    "pull_request": "update-or-create",
    "post_action": [
        "black"
    ],
    "disable_post_actions": false,
    "draft": true
}
```

### CODE REVIEW

File: docs/dev/arch_decision.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Architecture Decision Records
+
+The intention is to document deviations from a standard Model View Controller (MVC) design.
+
+!!! warning "Developer Note - Remove Me!"
+    Optional page, remove if not applicable.
+    For examples see [Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config/tree/develop/docs/dev/dev_adr.md) and [nautobot-plugin-reservation](https://github.com/networktocode/nautobot-plugin-reservation/blob/develop/docs/dev/dev_adr.md).
```
Comment (snaselj) [inline docs/dev/arch_decision.md:7]: Update or remove

File: docs/user/app_overview.md [hunk 2]
```diff
@@ -20,3 +25,14 @@ That being said, there is currently little reason to install this plugin by itse
...
+## Nautobot Features Used
+
+!!! warning "Developer Note - Remove Me!"
+    What is shown today in the Installed Plugins page in Nautobot. What parts of Nautobot does it interact with, what does it add etc. ?
+
+### Extras
+
+!!! warning "Developer Note - Remove Me!"
+    Custom Fields - things like which CFs are created by this app?
+    Jobs - are jobs, if so, which ones, installed by this app?
```
Comment (snaselj) [inline docs/user/app_overview.md:38]: Update or remove.

**Comment by snaselj** [comment]
This PR is ready for review.

cc.: @whitej6

File: docs/images/icon-nautobot-plugin-nornir.png
Comment (itdependsnetworks) [inline docs/images/icon-nautobot-plugin-nornir.png:1]: Where is this used? My gut would have said that we are not using this but the actual app icon.

File: docs/user/app_overview.md (hunk 2 above)
Comment (itdependsnetworks) [reply docs/user/app_overview.md:38]: I think we can remove.

File: docs/user/app_overview.md [hunk 3]
```diff
@@ -20,3 +25,14 @@ That being said, there is currently little reason to install this plugin by itse
 
 * itdependsnetworks
 * jeffkala
+
+## Nautobot Features Used
+
+!!! warning "Developer Note - Remove Me!"
+    What is shown today in the Installed Plugins page in Nautobot. What parts of Nautobot does it interact with, what does it add etc. ?
```
[the remark this answers is not in the document]
Comment (itdependsnetworks) [reply docs/user/app_overview.md:32]: I think we can remove.

File: docs/dev/arch_decision.md (hunk 1 above)
Comment (itdependsnetworks) [reply docs/dev/arch_decision.md:7]: I think we can remove.

File: nautobot_plugin_nornir/tests/test_basic.py [hunk 4]
```diff
@@ -3,6 +3,18 @@
 import os
 import toml
 
+from nautobot_plugin_nornir import __version__ as project_version
```
Comment (itdependsnetworks) [inline nautobot_plugin_nornir/tests/test_basic.py:6]: I don't think this is needed. I know we spoke about it before, but I still don't understand what this is testing now.

Comment (itdependsnetworks) [reply nautobot_plugin_nornir/tests/test_basic.py:6]: To clarify, I know what the test does, it just seems circular to me.

**Review by jeffkala — APPROVED** [review]
good once developer notes get removed.

File: docs/dev/arch_decision.md (hunk 1 above)
Comment (snaselj) [reply docs/dev/arch_decision.md]: Removed

File: docs/user/app_overview.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (snaselj) [reply docs/user/app_overview.md]: Removed

File: docs/user/app_overview.md (hunk 2 above)
Comment (snaselj) [reply docs/user/app_overview.md]: Removed

File: docs/images/icon-nautobot-plugin-nornir.png
Comment (snaselj) [reply docs/images/icon-nautobot-plugin-nornir.png:1]: Seems to be unused, removed the file.

**Comment by snaselj** [comment]
> good once developer notes get removed.

G2g from my perspective.

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**